toc/working-groups/app-runtime-interfaces.md

```diff
@@ -0,0 +1,174 @@
+# App Runtime Interfaces: Working Group Charter
+
+## Mission
+
+Provides APIs for the CF App Runtime and community clients for end users.
+
+
+## Goals
+
+- End users can build against a stable, reliable, performant and well documented CF API.
+- End users can build against a stable, reliable, performant and well documented higher-level CF API related services.
+- End users can choose from a range of CF API clients according to their needs: cli, UI and client libraries for selected programming languages.
+- Community Contributors and especially the App Runtime Deployments WG can integrate a tested CF API release into the different CF distributions.
+
+
+## Scope
+
+- Provide the community with regular releases of the CF API, clients and higher-level CF API related services.
+- Maintain public roadmaps for the CF API and the CF cli. Ensure a wise balance between stable APIs and clients, feature enhancements and deprecations.
+- Provide a CF API suitable for the different CF deployment scenarios: from small to very large foundations, VM- and k8s-based, support for major IaaS providers.
+- Provide the community with pipelines and test suites to validate functionality, compatibility and performance of the CF API.
+- Provide the community with technical API documentation, end user documentation and operator documentation.
+- Collaborate with the other Working Groups and evolve the cf-push experience.
+- Where it is appropriate, integrate higher-level API extensions into CF API directly.
+
+
+## Non-Goals
+
+- Provide the community with 'all' language specific CF clients libraries.
+
+
+## Open Questions
+
+- Should we split up this WG into a 'core interfaces' and 'extension' WG or try to manage the impressive long list of github projects in one WG? E.g.:
+  - 'core' App Runtime Interfaces WG: Buildpacks, CAPI, CLI
+  - App Runtime Extensions WG: Autoscaler, Java Tools, Multiapps, Notifications, Stratos, additional language clients (Python, golang)
+- Include https://github.com/cloudfoundry-community/cf-python-client as a second language binding that is under active development?
+- Ownership of CF API related test projects like capi-bara-tests, cf-performance-tests, cf-smoke-tests, cf-acceptance-tests: Interface or Deployments WG
+
+
+## Proposed Membership
+
+- Technical Lead(s): TBD
+- Execution Lead(s): TBD
+
+### Approvers by Area
+Autoscaler
+- tbd
+
+Buildpacks
```

Comment:
For Buildpacks (specifically the Java buildpack), I nominate @dmikusa-pivotal and @pivotal-david-osullivan.

I'll follow up with nominations for the other buildpacks.